Source/WebCore/rendering/RenderLayer.h

```diff
@@ -142,6 +143,7 @@ struct ScrollRectToVisibleOptions {
     const ScrollAlignment& alignY { ScrollAlignment::alignCenterIfNeeded };
     ShouldAllowCrossOriginScrolling shouldAllowCrossOriginScrolling { ShouldAllowCrossOriginScrolling::No };
     ScrollBehavior behavior { ScrollBehavior::Auto };
+    LayoutRect visibilityRect { LayoutRect() };
```

Comment:
LayoutRect has a default constructor. No need to provide an initializer.

Reply:
Fixed.